hw/chisel/src/main/scala/snax/streamer/DesignParams.scala

```diff
@@ -69,6 +73,30 @@ class StreamerParam(
     )
   }
 
+  // transpose parameters
+  if (hasTranspose) {
+    require(
+      readerNum == 2,
+      "Transpose only supports for gemm with 2 readers"
+    )
```

Comment:
why is this?

Reply:
Removed now! Please check: https://github.com/KULeuven-MICAS/snax_cluster/pull/323